backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -705,9 +714,9 @@ private void addCorrectionNote(
     }
   }
 
-  public Set<Observation> convertToAOEObservation(
+  public Set<Observation> convertToAOESymptomObservation(
       String eventId, Boolean symptomatic, LocalDate symptomOnsetDate) {
-    var observations = new HashSet<Observation>();
+    var observations = new LinkedHashSet<Observation>();
```

Comment:
changed to a LInkedHashSet to maintain the order for tests... please let me know if there's a better way 😅

Reply:
maybe we could use a different value of [JSONCompareMode](http://jsonassert.skyscreamer.org/javadoc/org/skyscreamer/jsonassert/JSONCompareMode.html) in our tests when we're asserting that expected equals actual. I don't think there's ever a case where we care about order in the FHIR bundle, so I think `NON_EXTENSIBLE` should work. we might even be okay with `LENIENT`, as long as we are good about adding test coverage for new fields

Reply:
I removed the `LinkedHashSet` and reran tests locally and fixed one test by making the comparison less strict... However, there's several tests in the  [`backend/src/test/java/gov/cdc/usds/simplereport/utils/BulkUploadResultsToFhirTest.java` ](https://github.com/CDCgov/prime-simplereport/pull/6608/files#diff-e5735ee044e3660c662afe1e85d477258f055c4fb3e5c62b0d2424d8b04b0a7d) file that rely on [comparing the values as strings](https://github.com/CDCgov/prime-simplereport/blob/625d61af13b59a4c102477279087cceceb98527b/backend/src/test/java/gov/cdc/usds/simplereport/utils/BulkUploadResultsToFhirTest.java#L343) so that fails 🤔